run/README.madwrf

```diff
@@ -0,0 +1,37 @@
+The MAD-WRF model is designed to improve the solar irradiance nowcast
```

Comment:
@pedro-jm 
This file should probably go into the WRF/doc directory

Reply:
I just moved the file to the WRF/doc directory